index.js

```diff
@@ -30,26 +26,26 @@ process.env.NODE_ENV =
         await node.start();
     } catch (e) {
         console.error(`Error occurred while start ot-node, error message: ${e}. ${e.stack}`);
-        console.error(`Trying to recover from older version`);
-        if (process.env.NODE_ENV !== NODE_ENVIRONMENTS.DEVELOPMENT) {
-            const rootPath = path.join(appRootPath.path, '..');
-            const oldVersionsDirs = (await fs.promises.readdir(rootPath, { withFileTypes: true }))
-                .filter((dirent) => dirent.isDirectory())
-                .map((dirent) => dirent.name)
-                .filter((name) => semver.valid(name) && !appRootPath.path.includes(name));
-
-            if (oldVersionsDirs.length === 0) {
-                console.error(
-                    `Failed to start OT-Node, no backup code available. Error message: ${e.message}`,
-                );
-                process.exit(1);
-            }
-
-            const oldVersion = oldVersionsDirs.sort(semver.compare).pop();
-            const oldversionPath = path.join(rootPath, oldVersion);
-            execSync(`ln -sfn ${oldversionPath} ${rootPath}/current`);
-            await fs.promises.rm(appRootPath.path, { force: true, recursive: true });
-        }
+        // console.error(`Trying to recover from older version`);
```

Comment:
What is plan with this part, to try to fix it later or?

Reply:
Yes, we need to decide on logic here, when we want to revert to previous version. This code is not working properly anyway so i just commented it out